Exercises/exercise_1_btc_transfer_cli.md

```diff
@@ -89,9 +89,9 @@ You can search it using `docker logs -f axelar-core 2>&1 | grep -e threshold`.
   ```
   axelard q ethereum sendCommand {commandID} {address of account that should send this tx}
   ```
-  e.g., for the testnet, we allow you to use our address as the sender = `0xE3deF8C6b7E357bf38eC701Ce631f78F2532987A`
+  e.g., for the testnet, we allow you to use our address as the sender = `0x608a44895ccb488ef764597cbc2aa7f780409ac2`
```

Comment:
* need to update in TESTNET RELEASE file.
* just link to the file here, so we don't need to modify all exercises each time it changes.

Reply:
This address has been replaced with the clef signer.